scripts/evergreen/deployments/test-app/templates/mongodb-enterprise-tests.yaml

```diff
@@ -175,6 +175,10 @@ spec:
     {{ end }}
     - name: ops_manager_version
       value: "{{ .Values.opsManagerVersion }}"
+    # TODO to be removed at public preview stage of community-search
```

Comment:
Is there a Jira ticket linked to those TODO actions?

Reply:
When we switch to the official MongoDB Search Community image repository in [CLOUDP-318799](https://jira.mongodb.org/browse/CLOUDP-318799) we'll revert this entire PR.